lib/pbench/test/unit/server/conftest.py

```diff
@@ -293,6 +303,33 @@ def provide_metadata(attach_dataset):
     Metadata.setvalue(dataset=test, key=Metadata.DELETION, value="2023-01-25")
 
 
+@pytest.fixture()
+def get_document_map(monkeypatch, attach_dataset):
```

Comment:
It's an interesting question; I have no idea what you'd call it, partly because the type changes: in the method signature, it's definitely a `Callable` with no parameters ... but within the body the same name becomes just a symbol bound to nothing or to whatever the referenced fixture returns. This is Python at its twistiest.

Reply:
Yeah, I have no idea what type a fixture is (I suppose I could code up a quick example or two and ask Python to tell me...), but I was thinking more along the lines of the fact that type hints aren't actually validated in anyway currently, so we could put basically anything we wanted there!  😈